webknossos/webknossos/dataset/dataset.py

```diff
@@ -64,19 +64,55 @@ class Dataset:
     which themselves can comprise multiple magnifications (webknossos.dataset.mag_view.MagView).
     """
 
-    def __init__(self, dataset_path: Union[str, Path]) -> None:
+    def __init__(
+        self,
+        dataset_path: Union[str, PathLike],
+        scale: Optional[Tuple[float, float, float]] = None,
```

Comment:
> Should we have a deprecation notice, in case it is unset?

Yes, I think, this would make it easier/clearer to remove the backwards-compatibility some day in the future.

Reply:
Added it :+1: